client/stylesheets/_toCat.scss

```diff
@@ -5800,7 +5800,9 @@ textarea {
     display: none !important;
 }
 
-
+.homeCardSize b {
+    font-weight: 400 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces